### ISSUE TITLE
Bootstrap stylesheets?!

### ISSUE THREAD

**Issue by zavalacl** [zavalacl/tc359_rails_final#3]
Where do I find these files!? I want to get rid of it!

line 15, /Users/zavalacl/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/bootstrap-sass-3.3.4.1/assets/stylesheets/bootstrap/_tables.scss 

th {
   text-align: left;
}

**Comment by chrisvfritz** [comment]
That's actually in the bootstrap-sass gem installed in your computer, being imported by [these two lines](https://github.com/zavalacl/tc359_rails_final/blob/master/app/assets/stylesheets/application.scss#L3-L4) in your application.scss file. Are you saying you want to get rid of Bootstrap in your project?

**Comment by zavalacl** [comment]
No, I just want to be able to edit those files, is there any way to do that because I can't seem to find them. Thanks!

**Comment by chrisvfritz** [comment]
Ah, I see. You'll actually never want to edit Bootstrap files directly. Otherwise, when you upgrade to a newer version of Bootstrap, all your changes will be overwritten and it would be difficult to remember all the changes you made to redo them.

In this case, since there's [no variable for table header alignment](http://getbootstrap.com/customize/#tables), I would write your own styles to override Bootstrap's. For example, in a file you could specify:

``` scss
th { 
  text-align: center;
}
```

Let me know if that makes sense. :smiley:
